### ISSUE TITLE
Question: Is there a way to visualize long files?

### ISSUE THREAD

**Issue by mkleinbort-ic** [pomber/git-history#193]
Some files in the code base I'm working on are 10k lines+

Is there any idea on how to handle this? Either collapsing regions that don't change, or offering a "wrap" version of the code base (ie. multiple columns)

Open to ideas.

**Comment by Adarshg315** [comment]
yes it would be good @mkleinbort-ic  to have a feature like that
